quicksilver/assetlist.json

```diff
@@ -473,6 +473,264 @@
+        {
+          "denom": "qpica",
+          "exponent": 12,
+          "aliases": []
+        }
+      ],
+      "base": "pqpica",
+      "name": "Quicksilver Liquid Staked PICA",
+      "display": "qpica",
+      "symbol": "qPICA",
+      "traces": [
+        {
+          "type": "liquid-stake",
+          "counterparty": {
+            "chain_name": "composable",
+            "base_denom": "ppica"
+          },
+          "provider": "Quicksilver"
+        }
+      ],
+      "logo_URIs": {
+        "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/quicksilver/images/qpica.png",
+        "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/quicksilver/images/qpica.svg"
+      },
+      "images": [
+        {
+          "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/quicksilver/images/qpica.png",
+          "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/quicksilver/images/qpica.svg",
+          "theme": {
+            "primary_color_hex": "#ff914d"
+          }
+        }
+      ],
+      "type_asset": "sdk.coin"
+    },
+    {
+      "description": "Quicksilver Liquid Staked INJ",
+      "denom_units": [
+        {
+          "denom": "qinj",
+          "exponent": 0,
+          "aliases": []
+        },
+        {
+          "denom": "qINJ",
+          "exponent": 18,
+          "aliases": []
+        }
+      ],
+      "base": "qinj",
+      "name": "Quicksilver Liquid Staked INJ",
+      "display": "qINJ",
+      "symbol": "qINJ",
+      "traces": [
+        {
+          "type": "liquid-stake",
+          "counterparty": {
+            "chain_name": "injective",
+            "base_denom": "inj"
+          },
+          "provider": "Quicksilver"
+        }
+      ],
+      "logo_URIs": {
+        "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/quicksilver/images/qinj.png",
+        "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/quicksilver/images/qinj.svg"
+      },
+      "images": [
+        {
+          "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/quicksilver/images/qinj.png",
+          "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/quicksilver/images/qinj.svg",
+          "theme": {
+            "primary_color_hex": "#ff914d"
+          }
+        }
+      ],
+      "type_asset": "sdk.coin"
+    },
+    {
+      "description": "Quicksilver Liquid Staked LUNA",
+      "denom_units": [
+        {
+          "denom": "uqluna",
+          "exponent": 0,
+          "aliases": []
+        },
+        {
+          "denom": "qluna",
+          "exponent": 6,
+          "aliases": []
+        }
+      ],
+      "base": "uqluna",
+      "name": "Quicksilver Liquid Staked LUNA",
+      "display": "qluna",
+      "symbol": "qLUNA",
+      "traces": [
+        {
+          "type": "liquid-stake",
+          "counterparty": {
+            "chain_name": "terra",
```

Comment:
I suspect this is meant to refer to terra2, since there is an IBC connection being registered between quicksilver and terra2 (and not terra classic)

Reply:
fixed.